src/Joomlatools/Joomla/Util.php

```diff
@@ -24,6 +24,27 @@ public static function getPlatformName()
         return static::isJoomlatoolsPlatform() ? 'Joomlatools Platform' : 'Joomla';
     }
 
+    /**
+     * Validate if the current working directory is a Standalone app
+     *
+     * @return bool
+     */
+    public static function isStandalone()
```

Comment:
@yiendos I'm not so sure about this check. I would do this different, the script is running standalone if it's not running in Joomla or in platform.

So maybe the solution is to not add a isStandalone() method but instead change the code, and do not do further installation processing by checking

`if (!Util::isJoomla() && !Util::isJoomlatoolsPlatform())`  

Thoughts?

Reply:
No makes sense and makes the code similar to other checks in the joomlatools-composer plugin